permission-portal-server/config.js

```diff
@@ -5,9 +5,15 @@ dotenv.config();
 
 const isProduction = process.env.NODE_ENV === "production";
 
-const connectionString = `postgresql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`
+const connectionString = `postgres://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`;
 
 export const pool = new Pool({
-  connectionString: isProduction ? process.env.DATABASE_URL : connectionString,
-  ssl: isProduction
-});
\ No newline at end of file
+  connectionString: isProduction
+    ? process.env.HEROKU_POSTGRESQL_BLUE_URL
```

Comment:
Heroku can provide multiple different POSTGRESQL_<COLOR> urls, but the main one will be DATABASE_URL. We should probably use that instead of this.